aten/src/ATen/native/EmbeddingBag.cpp

```diff
@@ -25,8 +25,9 @@ namespace native {
 
 static void make_offset2bag(const Tensor &offsets, const Tensor &indices,
                             Tensor &offset2bag) {
-  offset2bag.index_fill_(0, offsets, 1); // offset2bag = [1 0 1 0 1]
-  offset2bag[0] = 0;                     // offset2bag = [0 0 1 0 1]
+  offset2bag.index_add_(
```

Comment:
was this change reviewed by someone? @colesbury did you review it?

Reply:
I will remove this.

For context, it seems that this slipped through from D7875031, which landed before the last github->fbcode sync and therefore didn't take into account PR 5725. There is a new PR 7389 which should contain all of D7875031 and account for PR 5725, and none of the changes in D7875031 should appear in this sync. See my merge note https://github.com/pytorch/pytorch/pull/7451/commits/d6790722c98f8b5fb5fd80838b8f1cd44649a494